solutions/platinum/usaco-1332.mdx

```diff
@@ -0,0 +1,265 @@
+---
+id: usaco-1332
+source: USACO Platinum 2023 March
+title: Pareidolia
+author: Aakash Gokhale
+---
+
+## Explanation
+
+Let's first try to solve the problem without updates on a single string.
+Notice that it is always optimal to take the earliest $b$ and then the earliest $e$ and then the earliest $i$ after that and so on because it maximizes the number of places you can search to find the next letter of "bessie".
```

Comment:
```suggestion
Notice that it is always optimal to take the earliest $b$ and then the earliest $e$ and then the earliest $s$ after that and so on because it maximizes the number of places you can search to find the next letter of "bessie".
```